### PR TITLE
Release candidate for version 2.3.1

### PR DESCRIPTION
**Pull request by amirhosv**
*Description of changes:*

Release candidate for version 2.3.1

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by brian-jarvis-aws — COMMENTED** [review]
Can you include the "main changes" note from the [AWS-LC 1.15 notes](https://github.com/aws/aws-lc/releases/tag/v1.15.0) so that it's more obvious what the benefit is?

**Comment by amirhosv** [comment]
> Can you include the "main changes" note from the [AWS-LC 1.15 notes](https://github.com/aws/aws-lc/releases/tag/v1.15.0) so that it's more obvious what the benefit is?

I just mentioned the change in LC that is relevant to ACCP.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.3.1
+
+### Patches
+* Use AWS-LC [v1.15.0](https://github.com/aws/aws-lc/releases/tag/v1.15.0) for ACCP
```
Comment (WillChilds-Klein) [inline CHANGELOG.md:6]: do we need to update the submodule commit ID?

**Review by brian-jarvis-aws — DISMISSED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (amirhosv) [reply CHANGELOG.md:6]: Done. It turns out that AWS-LC now needs Go to build and our CI task for the legacy build did not have it.

Comment (WillChilds-Klein) [reply CHANGELOG.md:6]: oh, that's annoying. thanks for resolving that issue!

**Review by WillChilds-Klein — APPROVED** [review]

**Pull request merged**